packages/earth-admin/src/utils/validations.ts

```diff
@@ -22,7 +22,7 @@
  * Show error if string contains special characters.
  */
 export const noSpecialChars = (value: string): boolean => {
-  const regex = RegExp(/^(?:[A-Za-z]+)(?:[A-Za-z0-9 _]*)$/);
+  const regex = RegExp('^[a-z0-9](-?[a-z0-9])*$');
```

Comment:
This breaks the validation on New Place -> Name field where the user should be able to add spaces.
Leave noSpecialChars validation as it is and create another one for your use case